dune-project

```
@@ -7,18 +7,25 @@
 (source
  (github ocurrent/opam-ci-check))
 
-(authors "Author Name")
+(authors
+ "Puneeth Chaganti <punchagan@muse-amuse.in>"
+ "Shon Feder <shon.feder@gmail.com>")
 
-(maintainers "Maintainer Name")
+(maintainers
+ "Puneeth Chaganti <punchagan@muse-amuse.in>"
+ "Shon Feder <shon.feder@gmail.com>")
 
-(license LICENSE)
+; TODO: Add a license field
+;(license LICENSE)
 
-(documentation https://url/to/documentation)
+(documentation
+ https://www.ocurrent.org/opam-ci-check/opam-ci-check/index.html)
 
 (package
  (name opam-ci-check)
- (synopsis "A short synopsis")
- (description "A longer description")
+ (synopsis "Opam Repo CI in your CLI")
```

I like the phrasing here :D that said I'm not sure what "your" or "my" CLI is. Could it be something like

```suggestion
 (synopsis "CLI driving the opam repo CI")
```

or 

```suggestion
 (synopsis "Opam repo CI's CLI")
```

---

Ah, I guess I meant `Opam Repo CI in your terminal (not CLI)`.